api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -28,11 +29,11 @@
   // The email address that can be used to contact the user.
   private String contactEmail;
   private DataAccessLevel dataAccessLevel;
-  private String fullName;
   private String givenName;
```

Comment:
Do we really need to store these fields now that they are available via Google's Directory Service API?

Reply:
I think we will want some of this information locally in order to facilitate the storefront researcher directory in future (where we'll show lists of researchers with their name and contact information.)

The alternative is doing a join between our database and the data in directory service... which sounds painful.

Having the data in two places seems OK to me.

Reply:
Also: in future we may want to support bring-your-own-account. At that point, we won't be able to use directory service for listing researchers anymore.